### PR TITLE
Fix static scan issues

### PR DESCRIPTION
**Pull request by xianju6x**

### CODE REVIEW

File: installer.c [hunk 1]
```diff
@@ -279,6 +279,9 @@ static EFI_STATUS installer_flash_big_chunk(EFI_FILE *file, UINTN *remaining_dat
 	EFI_STATUS ret = EFI_INVALID_PARAMETER;
 	UINTN payload_size, read_size, already_read, ckh_blks, data_size;
 	const UINTN MAX_DATA_SIZE = dl->max_size - offsetof(flash_buffer_t, ckh_data);
+	if (fb->sph.blk_sz == 0) {
+        return EFI_INVALID_PARAMETER;
```
Comment (James-Zzz) [inline installer.c:283]: Add indent here

File: libkernelflinger/slot_avb.c [hunk 2]
```diff
@@ -198,21 +198,18 @@ static EFI_STATUS select_highest_priority_slot(void)
...
-		    slot->successful_boot == 0) {
-			ret = disable_slot(slot, TRUE);
-			if (EFI_ERROR(ret))
-				return ret;
-		}
-
-		cur_suffix = suffixes[slot - slots];
-		break;
-	}
+    slot = highest_priority_slot();
```
Comment (James-Zzz) [inline libkernelflinger/slot_avb.c:201]: add indent from line 201 to line 212

File: libkernelflinger/ui_textarea.c [hunk 3]
```diff
@@ -96,7 +96,9 @@ static void ui_textarea_copy_char(unsigned char *src_p, UINTN src_row_bytes,
 				  EFI_GRAPHICS_OUTPUT_BLT_PIXEL *color)
 {
 	int i, j;
-
+    if (!color) {
```
Comment (James-Zzz) [inline libkernelflinger/ui_textarea.c:99]: Add indent from line 99 to line 101

File: libkernelflinger/vars.c [hunk 4]
```diff
@@ -867,7 +868,9 @@ char *get_serial_number(void)
 
 	return serialno;
 bad:
-	strncpy_s((CHAR8 *)serialno, sizeof(serialno), (CHAR8 *)"00badbios00badbios00", SERIALNO_MAX_SIZE);
+	ret = strncpy_s((CHAR8 *)serialno, sizeof(serialno), (CHAR8 *)"00badbios00badbios00", SERIALNO_MAX_SIZE);
+    if (EFI_ERROR(ret))
```
Comment (James-Zzz) [inline libkernelflinger/vars.c:872]: Add indent at line 872 and 873

**Review by James-Zzz — COMMENTED** [review]
Please add sign-off and Tracked on in commit message.
We are using this OAM-105551 to track this commit

**Review by James-Zzz — APPROVED** [review]
LGTM

**Review by kwang13 — APPROVED** [review]

File: libkernelflinger/slot_avb.c [hunk 5]
```diff
@@ -198,22 +198,19 @@ static EFI_STATUS select_highest_priority_slot(void)
...
-		slot = highest_priority_slot();
-		if (!slot)
-			return EFI_NOT_FOUND;
-
-		if (slot->tries_remaining == 0 &&
-		    slot->successful_boot == 0) {
-			ret = disable_slot(slot, TRUE);
-			if (EFI_ERROR(ret))
-				return ret;
-		}
```
Comment (GangSecurity) [inline libkernelflinger/slot_avb.c:211]: any reason to remove the while loop?

File: libedk2_tpm/Tpm2Capability.c [hunk 6]
```diff
@@ -566,6 +566,7 @@ Tpm2TestParms (
     default:
       return EFI_INVALID_PARAMETER;
     }
+    break;
```
Comment (GangSecurity) [inline libedk2_tpm/Tpm2Capability.c:569]: Code is the same as https://github.com/tianocore/edk2/blob/master/SecurityPkg/Library/Tpm2CommandLib/Tpm2Capability.c
Could you double confirm？

**Pull request merged**